cinema/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+# Create your views here.
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movie = Movie.objects.all()
+        serializers = MovieSerializer(movie, many=True)
+        return Response(serializers.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+
+        if serializer.is_valid():

```

Comment:
```suggestion
        serializer.is_valid(raise_exception=True):

```
This will avoid handling the exception manually